### PR TITLE
add libpod.conf man page (closes #537)

### LINKED ISSUE #537: Missing man page for libpod.conf
We need to document content in libpod.conf with a man page

### PR DESCRIPTION
**Pull request by nathwill**
saw this todo open, thought i'd take a whack at it. anything i should add (e.g. SEE ALSO section, or...)?

### CODE REVIEW

File: docs/libpod.conf.5.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+% libpod.conf(5) library for running OCI-based containers in Pods
+% Nathan Williams
+% APRIL 2018
+
+## NAME
```
Comment (rhatdan) [inline docs/libpod.conf.5.md]: Should add default path.

**Comment by rhatdan** [comment]
You will have to fix the Makefile to install into man5 as well as man1.  Probably want to rename all of the existing man pages to 1.md files.

**Comment by mheon** [comment]
@nathwill Missing a signoff on the commit, can you sign with -s and force-push?

**Comment by rhatdan** [comment]
LGTM
@mheon @umohnani8 @baude @TomSweeneyRedHat PTAL

**Comment by mheon** [comment]
LGTM
@rh-atomic-bot r+

**Comment by rhatdan** [comment]
HOMU seems to be hung.

**Pull request merged**